README.md

```diff
@@ -168,6 +168,35 @@ Dato::Client.new.items.update(attributes: { title: 'Hello world' }, item_id: '12
 Dato::Client.new.items.destroy(item_id: '123')
 ```
 
+## File upload
+
+Dato Rails also supports file uploads. 
+These can be created either from a local file or from a url. 
+Basically all file types are supported, as long as they are valid in the CMS.
+In addition to the binary file, also attributes and metadata can be uploaded. 
+Both metadata and attributes are optional.
+
+### Upload from Url
+```ruby
+Dato::Client.new.uploads.create_from_url(url: 'https://picsum.photos/seed/picsum/200/300')
+Dato::Client.new.uploads.create_from_url(url: 'https://picsum.photos/seed/picsum/200/300', attributes:, meta:)
+```
+
+### Upload from Local File
+```ruby
+file = File.open('image.png')
+
+Dato::Client.new.uploads.create_from_file(path_to_file: file.path)
+Dato::Client.new.uploads.create_from_file(path_to_file: file.path, attributes:, meta:)
+```
+
+### Optional: Attributes and metadata
+
+```ruby
+attributes = { author: 'Dato Rails' }
+meta = { en: { title: 'Static Random Image', alt: 'some caption text' } }
```

Comment:
is there a restriction about what attributes and meta can be sent? or is there a specific format should be used?
Maybe we can link to the official docs.